schema/tables/icloud_private_relay.yml

```
@@ -0,0 +1,7 @@
+name: icloud_private_relay
+description: Whether (iCloud Private Relay)[https://support.apple.com/en-us/HT212614] is turned on.
```

```suggestion
description: Whether (iCloud Private Relay)[https://support.apple.com/en-us/HT212614] is enabled.
```
Not sure if "turned on" is the best term, sounds funny 😅